requirements.txt

```diff
@@ -1,3 +1,3 @@
-docutils<=0.14
+docutils<=0.16
```

Comment:
It seems odd that we require docutils `<=` a particular version. IIRC this was to work around some issue in the past. But we have been using `==` for these versions more broadly. Can we use `==` here?

Reply:
Seems reasonable to me!